For not yet understood reasons the shlibs for ldap-utils cannot be computed
since it fails to find the symbols of libldap-2.4.2.

Reevaluate this later once UCS 4.0 is fully build.

UCS Bug #33835

diff -aur openldap-2.4.35.orig/debian/control openldap-2.4.35/debian/control
--- openldap-2.4.35.orig/debian/control	2014-01-25 09:52:45.892000000 +0100
+++ openldap-2.4.35/debian/control	2014-01-25 11:23:17.772000000 +0100
@@ -69,7 +69,7 @@ Description: transitional package for slapd-contrib
 
 Package: ldap-utils
 Architecture: any
-Depends: ${shlibs:Depends}, libldap-2.4-2 (= ${binary:Version}), ${misc:Depends}
+Depends: libldap-2.4-2 (= ${binary:Version}), ${misc:Depends}
 Recommends: libsasl2-modules
 Suggests: libsasl2-modules-gssapi-mit | libsasl2-modules-gssapi-heimdal
 Conflicts: umich-ldap-utils, openldap-utils, ldap-client
diff -aur openldap-2.4.35.orig/debian/rules openldap-2.4.35/debian/rules
--- openldap-2.4.35.orig/debian/rules	2014-01-25 09:52:46.004000000 +0100
+++ openldap-2.4.35/debian/rules	2014-01-25 11:22:52.148000000 +0100
@@ -210,6 +210,9 @@ override_dh_installdeb:
 	perl -w debian/dh_installscripts-common -p slapd
 endif
 
+override_dh_shlibdeps:
+	dh_shlibdeps -Xdebian/ldap-utils
+
 override_dh_auto_clean:
 	dh_auto_clean
 	# Update translation templates for debconf
